src/pages/Cart/CartList.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCart } from "@/contexts/CartContext";
+import { S } from "./Cart.styles";
+import { useCartProducts } from "@/hooks/useCartProducts";
+import CartItemRow from "./CartItemRow";
+import CartTotals from "./CartTotals";
+
+const shippingPrice = 100;
+
+export default function CartList() {
+  const { updateItemQuantity, clear } = useCart();
+  const { combinedItems, totalPrice, isFetching } = useCartProducts();
```

Comment:
`useFetch` exposes error, but CartList doesn’t show it.


```suggestion
  const { combinedItems, totalPrice, isFetching, error } = useCartProducts();
  if (error) return <p>Failed to load cart. Please try again.</p>;
```